app/serializers/auction_serializer.rb

```diff
@@ -42,6 +43,14 @@ def winning_bid
 
   private
 
+  def veiled_bids
```

Comment:
note: had to add this because we used to pass in a presenter auction to this serializer but now we are just passing `auction`